Service/DRCService.php

```diff
@@ -232,13 +234,27 @@ public function downloadInhoudHandler(array $data, array $configuration): array
         $path = $this->data['path'];
 
         $objectEntity = $this->entityManager->getRepository('App:ObjectEntity')->find($path['id']);
+        $file = $this->entityManager->getRepository('App:File')->find($path['id']);
         if (
-            $objectEntity instanceof ObjectEntity &&
+            $file instanceof File
+        ) {
+            $this->data['response'] = new Response(\Safe\base64_decode($file->getBase64()), 200, ['content-type' => $file->getMimeType()]);
+        } else if (
+            $objectEntity instanceof ObjectEntity === true &&
             $objectEntity->getEntity()->getId()->toString() == $configuration['enkelvoudigInformatieObjectEntityId']
         ) {
-            $this->data['response'] = new Response(\Safe\base64_decode($objectEntity->getValueObject('inhoud')->getFiles()->first()->getBase64()), 200, ['content-type' => $objectEntity->getValueObject('inhoud')->getFiles()->first()->getMimeType()]);
+            $criteria = Criteria::create()->orderBy(['dateCreated' => Criteria::ASC]);
+            switch(isset($this->data['query']['versie']) === true) {
```

Comment:
Why use a switch? this looks like something we can just do with an if statement instead?

Reply:
Started out to try something more complex, will collapse it into an if statement